sdk/core/azure-core/CHANGELOG.md

```diff
@@ -5,11 +5,14 @@
 ### Features Added
 
 - Request logs to now include the `accept-range`, `content-range`, `range`, `WWW-Authenticate`, `x-ms-date`, `x-ms-error-code`, `x-ms-range`, and `x-ms-version` headers.
+- Added default constructor, `Parse()`, and equality comparison operators to `Azure::Core::Uuid`.
```

Comment:
Who do we expect to use the Parse method today (in the SDK/end user), or are we adding it preemptively for a near-future/known use case?